suites/reef/upgrades/tier-2-upgrade-rhcs-5x-to-7x-standby-mgr.yaml

```diff
@@ -0,0 +1,116 @@
+#=================================================================================================================
+# Test-Case: Upgrade cluster by having two standby mgr daemons already upgraded and the active mgr daemon removed.
+#
+# Cluster Configuration:
+#    conf/reef/upgrades/1admin-3node-1client-upgrade.yaml
+#=================================================================================================================
+
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: Deploy RHCS 5x cluster using cephadm
+      desc: Bootstrap and deployment services with label placements
+      polarion-id: CEPH-83573777
+      module: test_cephadm.py
```

Comment:
can't we use - https://github.com/red-hat-storage/cephci/blob/master/tests/cephadm/test_cephadm_bootstrap.py ?

Reply:
Hi Vaibhav,

We have to use old one only, Since feature applying spec along with bootstrap fixed in "qunicy", I current suite we have deploy pacific and upgrade